### PR TITLE
Diff should implement Serializable otherwise I am not able to include…

### PR DESCRIPTION
**Pull request by klearchos**
… it inside a DTO object that already implements Serializable.

### CODE REVIEW

File: javers-core/src/main/java/org/javers/core/diff/Diff.java [hunk 1]
```diff
@@ -24,7 +25,7 @@
  *
  * @author bartosz walacik
  */
-public class Diff {
+public class Diff implements Serializable {
```
Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/core/diff/Diff.java]: thanks for contributing to JaVers,
what about Diff components? Change, Atomic, etc. Think we need a simple test which proofs that Diff serialization works.
You could add it to JaversDiffE2ETest

**Comment by xeraa** [comment]
Any update on this? I'm also having issues since Diff isn't serializable

**Comment by bartoszwalacik** [comment]
@xeraa, as you see, @klearchos have created this Pull Request but he have abandoned the work ... 
you could continue this PR if you want, it's really easy.
Just add `serializable` for few classes and write an integration test

**Comment by pawelszymczyk** [comment]
I polished it a bit in https://github.com/javers/javers/pull/269

**Pull request merged**